components/script/dom/webglframebuffer.rs

```diff
@@ -190,7 +190,7 @@ impl WebGLFramebuffer {
             // Note, from the GLES 2.0.25 spec, page 113:
             //      "If texture is zero, then textarget and level are ignored."
             Some(texture) => {
-                *binding.borrow_mut() = Some(WebGLFramebufferAttachment::Texture(JS::from_ref(texture)));
+                *binding.borrow_mut() = Some(WebGLFramebufferAttachment::Texture(JS::from_ref(texture), level));
```

Comment:
should we move this after the level validation? Otherwise in your next patch we can't be convert it to an unsigned integer right away.